src/SearchParameters/index.js

```diff
@@ -312,6 +312,30 @@ SearchParameters.prototype = {
     } );
   },
   /**
+   * Get the list of conjunctive refinements for a single facet
+   * @param {string} facetName
+   * @return {string[]} list of refinements
+   */
+  getConjunctiveRefinements : function( facetName ) {
+    return this.facetsRefinements[ facetName ];
```

Comment:
I would append `|| []` to ease its usage. What do you think? (same for all others)